dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -947,15 +947,15 @@ def _estimate_mode_from_histogram(self) -> List[float]:
         mode = (
             bin_edges[highest_idxs] + bin_edges[highest_idxs + 1]  # type: ignore
         ) / 2
-        return mode.tolist()
+        return cast(List[float], mode.tolist())
 
     def _estimate_stats_from_histogram(self) -> float:
         # test estimated mean and var
         bin_counts = self._stored_histogram["histogram"]["bin_counts"]
         bin_edges = self._stored_histogram["histogram"]["bin_edges"]
         mids = 0.5 * (bin_edges[1:] + bin_edges[:-1])
         mean = np.average(mids, weights=bin_counts)
-        var = np.average((mids - mean) ** 2, weights=bin_counts)
+        var: float = np.average((mids - mean) ** 2, weights=bin_counts)
```

Comment:
as above

Reply:
This will end up being a np.float64. I changed the type annotation and method return type to reflect this.